Task5/src/FilterTime.java

```diff
@@ -0,0 +1,24 @@
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+public class FilterTime implements Filter {
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) throws IOException, ServletException {
+        long start = System.nanoTime();
+        filterChain.doFilter(servletRequest, servletResponse);
+        long end = System.nanoTime();
+        long time = Math.round(end - start);
```

Comment:
What is being rounded here?) `Math.round()` is for retrieving an integer value from a floating-point value, but that's not the case